.github/workflows/lint_on_push_or_pull.yml

```diff
@@ -49,15 +73,14 @@ jobs:
         with:
           python-version: ${{ matrix.python-version }}
 
-      - name: Install Poetry
-        uses: snok/install-poetry@v1
-        with:
-          version: ${{ env.POETRY_VERSION }}
-
-      - name: Install pre-commit
+      - name: Install black
         shell: bash
-        run: poetry install --with dev
```

Comment:
TODO (for next PR: make new workflow, installing poetry for each python package and run unit tests)